### ISSUE TITLE
ABAP functions not detected in SELECT

### ISSUE THREAD

**Issue by Kaszub09** [abaplint/abaplint#3348]
Something like
```abap
  SELECT object FROM tadir WHERE @( strlen( 'asd' ) ) = 2 INTO TABLE @DATA(TEST).
```
triggers `parser_error` with `Statement does not exist in ABAPv750(or a parser error), "SELECT" `, but there are no syntax errors inside SAP and code executes.
Same for e.g. `@( lines( ... ) ) > 0` (so maybe any `@( [abap_statement] )`?

I have `"syntax": {  "version": "v750"` in abap lint file config.
![obraz](https://github.com/abaplint/abaplint/assets/34368953/4ba8cf02-b5a3-454d-b7c7-7c5c84543808)

**Comment by larshp** [comment]
@Kaszub09 try again, 2.110.6 published

**Comment by Kaszub09** [comment]
It passes now, thanks.
